test/contrib/rdbms_test.py

```diff
@@ -0,0 +1,251 @@
+# Copyright 2012-2015 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import luigi
+import luigi.contrib.redshift
+import mock
+
+import unittest
+
+
+# Fake AWS and S3 credentials taken from `../redshift_test.py`.
+AWS_ACCESS_KEY = 'key'
+AWS_SECRET_KEY = 'secret'
+
+AWS_ACCOUNT_ID = '0123456789012'
+AWS_ROLE_NAME = 'MyRedshiftRole'
+
+BUCKET = 'bucket'
+KEY = 'key'
+
+
+class DummyS3CopyToTableBase(luigi.contrib.redshift.S3CopyToTable):
+    # Class attributes taken from `DummyPostgresImporter` in
+    # `../postgres_test.py`.
+    host = 'dummy_host'
+    database = 'dummy_database'
+    user = 'dummy_user'
+    password = 'dummy_password'
+    table = luigi.Parameter(default='dummy_table')
+    columns = luigi.TupleParameter(
+        default=(
+            ('some_text', 'varchar(255)'),
+            ('some_int', 'int'),
+        )
+    )
+
+    copy_options = ''
+    prune_table = ''
+    prune_column = ''
+    prune_date = ''
+
+    def s3_load_path(self):
+        return 's3://%s/%s' % (BUCKET, KEY)
+
+
+class DummyS3CopyToTableKey(DummyS3CopyToTableBase):
+    aws_access_key_id = AWS_ACCESS_KEY
+    aws_secret_access_key = AWS_SECRET_KEY
+
+
+# Using Redshift as the test bed since Redshift implements RDBMS. Could have
+# opted for PSQL but I'm less familar with that contrib.
```

Comment:
Perhaps make this comment a docstring to add emphasis. I almost asked why redshift was being used, but then i saw this comment